modules/installing-oadp-rosa-sts.adoc

```diff
@@ -0,0 +1,253 @@
+  name: ${CLUSTER_NAME}-oadp
+  provider: aws
+  region: $REGION
+EOF
+----
+// bringing over from MOB docs
+. Check your application's storage default storage class by entering the following command:
++
+[source,terminal]
+----
+$ oc get pvc -n <namespace>
+----
+
++
+.Example output:
+
++
+[source,terminal]
+----
+NAME     STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+applog   Bound    pvc-351791ae-b6ab-4e8b-88a4-30f73caf5ef8   1Gi        RWO            gp3-csi        4d19h
+mysql    Bound    pvc-16b8e009-a20a-4379-accc-bc81fedd0621   1Gi        RWO            gp3-csi        4d19h
+----
+
+
+. Get storage class by running the following command:
++
+[source,terminal]
+----
+$ oc get storageclass
+----
+
++
+.Example output:
++
+[source,terminal]
+----
+NAME                PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+gp2                 kubernetes.io/aws-ebs   Delete          WaitForFirstConsumer   true                   4d21h
+gp2-csi             ebs.csi.aws.com         Delete          WaitForFirstConsumer   true                   4d21h
+gp3                 ebs.csi.aws.com         Delete          WaitForFirstConsumer   true                   4d21h
+gp3-csi (default)   ebs.csi.aws.com         Delete          WaitForFirstConsumer   true                   4d21h
+----
+
+The following storage classes will work:
+
+* gp3-csi
+* gp2-csi
+* gp3
+* gp2
+
+If the application or applications that are being backed up are all using persistent volumes (PVs) with Container Storage Interface (CSI), it is advisable to include the CSI plugin in the OADP DPA configuration.
+
+. Create the `DataProtectionApplication` resource, to configure the connection to the storage where the backups and volume snapshots are stored:
+
+// Are there any callouts that you want added to the following
+// Do you prefer the cat command adding or removing? I am kind of agnostic
+.. Deploy a Data Protection Application for CSI volumes only by entering the following command:
++
+[source,terminal]
+----
+$ cat << EOF | oc create -f -
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: ${CLUSTER_NAME}-dpa
+  namespace: openshift-adp
+spec:
+  backupImages: false
+  features:
+    dataMover:
+      enable: false
+  backupLocations:
+  - bucket:
+      cloudStorageRef:
+        name: ${CLUSTER_NAME}-oadp
+      credential:
+        key: credentials
+        name: cloud-credentials
+      default: true
+      config:
+        region: ${REGION}
+  configuration:
+    velero:
+      defaultPlugins:
+      - openshift
+      - aws
+      - csi
+    restic:
+      enable: false
+EOF
+----
+
+// . Create the `DataProtectionApplication` resource, which is used to configure the connection to the storage where the backups and volume snapshots are stored:
+
+.. Deploy a Data Protection Application for CSI or non-CSI volumes by entering the following command:
```

Comment:
it should be point 1.b, right ? currently both points are rendered as a in preview.